### ISSUE TITLE
add read me 

### ISSUE THREAD

**Issue by masihfathi** [skeeks-cms/app-basic#3]
could you plz add read me how to use this template and can we use this template instead of yii2 basic template?

**Comment by skeeks-semenov** [comment]
Yes, Put here like this
```

# Download latest version of composer
curl -sS https://getcomposer.org/installer | COMPOSER_HOME=.composer php

# Installing the base project SkeekS CMS
COMPOSER_HOME=.composer php composer.phar create-project --prefer-dist --stability=dev skeeks/app-basic demo.ru
# Going into the project folder
cd demo.ru

#Edit the file to access the database, it is located at common/config/db.php

#Installation of ready-dump
php yii dbDumper/mysql/restore

```

**Comment by masihfathi** [comment]
can we use our own developed yii2 extensions in this template? and if we want to extend your modules how can we do that for exmple user module/

**Comment by skeeks-semenov** [comment]
Of course! Any solutions for yii2 are compatible with this template. And besides that, there are CMS add-ons that allow you to write solutions written according to defined rules immediately into a common system.

https://github.com/skeeks-cms/cms-log-db-target 
For example, if you install this solution, it will immediately appear in the cms administration system

**Comment by masihfathi** [comment]
thanks for your great work in the skeeks, in our team we decided to change our strategy to use ready made solution for routin work and develop other things carefully for the project cause using diffrent strategy for work make problem for example using wordpress near or yii2 app made sync problems and a lot of problems, but using a uniform template which has usual need for the site like your system(blogging, usal shop, create page, forms) and add extra features to this  template make a lot of sense.
another suggestion i work a lot on this extension
https://github.com/pceuropa/yii2-forms
and add a lot of features like editing submitted form, view submited form, deleting and add after submit events for payments and... if you want i can give source code to use in your cms. it is really like joomla rsform.

**Comment by skeeks-semenov** [comment]
Yes fine. I watched the demo, very interesting.

**Comment by masihfathi** [comment]
in this week i will add complete extension to github and send you.

**Comment by masihfathi** [comment]
hi again, i added the completed drag and drop form builder (as i promised) "https://github.com/masihfathi/yii2-drag-drop-forms"

**Comment by masihfathi** [comment]
@skeeks-semenov hi, did you see the extension?

**Comment by skeeks-semenov** [comment]
@masihfathi I'll try to look at the weekend

**Comment by masihfathi** [comment]
?
